### PR TITLE
Added GitHub Actions to automate packaging

### PR DESCRIPTION
**Pull request by penev92**
Same as https://github.com/OpenRA/Eluant/pull/5, https://github.com/OpenRA/OpenAL-CS/pull/4 and https://github.com/OpenRA/SDL2-CS/pull/6:

Adding 2 workflows to automate compiling of native dependencies and packaging it all into a NuGet package and publishing that.

The native binary compilation steps are taken from @pchote's https://github.com/pchote/native-dependencies

For results of a successful run see:
https://github.com/penev92/FreeType/actions/runs/4870852735
https://www.nuget.org/packages/OpenRA-Freetype6/1.0.10-a2

P.S.: I added a few improvements over the previous PRs which I plan on porting back later to the other 3 repositories.

### CODE REVIEW

**Comment by penev92** [comment]
Pending update do FreeType 2.13.0.

**Review by Mailaender — APPROVED** [review]
Tested on Linux x64

**Comment by penev92** [comment]
Updated to FreeType 2.13.0:
https://github.com/penev92/FreeType/actions/runs/4874628343
https://www.nuget.org/packages/OpenRA-Freetype6/1.0.10-a3

**Comment by penev92** [comment]
Trimmed down Linux-x64 dependencies.
https://github.com/penev92/FreeType/actions/runs/4880674381
https://www.nuget.org/packages/OpenRA-Freetype6/1.0.10-a5

**Comment by Mailaender** [comment]
Didn't break it.

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+; Unix-style newlines
+[*]
+end_of_line = LF
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+; 4-column tab indentation
+[*.yml]
+indent_style = space
```
Comment (abcdefg30) [inline .editorconfig:13]: Space? (If wanted then change the comment.)

File: .github/workflows/build-natives.yml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+                sudo apt-get install mingw-w64
+
+            - name: Compile natives
+              run: |
+                curl -s -L -O https://github.com/freetype/freetype/archive/refs/tags/${FREETYPE_VERSION}.zip
+                unzip ${FREETYPE_VERSION}.zip
+                mkdir freetype-${FREETYPE_VERSION}/build
+                cp win64.patch freetype-${FREETYPE_VERSION}/win64.patch
+                cd freetype-${FREETYPE_VERSION}
+                patch -p0 < win64.patch
```
Comment (pchote) [inline .github/workflows/build-natives.yml:35]: `patch -p0 < ../win64.patch` and remove the `cp` above?

**Comment by penev92** [comment]
Updated and packaged version `1.0.10-a6`.

**Comment by penev92** [comment]
This PR would be as good a place as any to add the proposed CI build from https://github.com/OpenRA/SDL2-CS/pull/7, but IMO we should have it run the `build-wrapper` and `build-natives` workflows rather than `build-nuget`, because that one will try to package and publish a NuGet package to nuget.org on every open PR and every push to a PR.

**Comment by penev92** [comment]
> Added workflow run for PRs as a CI check

**Pull request merged**